build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if command -v dpkg-query -l zip
```

Comment:
We should use something OS agnostic, like `which zip`

Reply:
This works with on all linux distributions and mac os as well.